### PR TITLE
8282937: Write a regression test for JDK-4820080

### PR DESCRIPTION
**Pull request by manukumarvs**
Write a regression test for [JDK-4820080](https://bugs.openjdk.java.net/browse/JDK-4820080) : RFE: Cannot Change the JSplitPane Divider Color while dragging

Issue(as per bug description):
I want to change the color of the JSplitPane divider. I
opened the code and found that, the color is hard coded to
darkGray while one is dragging the splitDivider. I have an
application where the components on SplitPane have black
background, So when i drag I can not see the split getting
dragged. Can some one fix this, by externelising the
SplitDivider color so that it can be changed at run time.

Testing:
Java 1.4.1 -> Test Failed.
$ ./j2sdk1.4.1/bin/java JSplitPaneDragColorTest
Metal uses correct drag color false
CDE/Motif uses correct drag color false
Windows uses correct drag color false
Test Failed.

Java 1.5.0 -> Test Failed.
$ ./jdk1.5.0/bin/java JSplitPaneDragColorTest
Metal uses correct drag color true
Windows uses correct drag color true
Windows Classic uses correct drag color true
Test Passed.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8282937](https://bugs.openjdk.java.net/browse/JDK-8282937): Write a regression test for JDK-4820080

### Reviewers
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7803/head:pull/7803` \
`$ git checkout pull/7803`

Update a local copy of the PR: \
`$ git checkout pull/7803` \
`$ git pull https://git.openjdk.java.net/jdk pull/7803/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7803`

View PR using the GUI difftool: \
`$ git pr show -t 7803`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7803.diff">https://git.openjdk.java.net/jdk/pull/7803.diff</a>

</details>

### CODE REVIEW

File: test/jdk/javax/swing/JSplitPane/4820080/JSplitPaneDragColorTest.java [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    private static JFrame frame;
+    private static JSplitPane pane;
+    private static Robot robot;
+
+    public static void main(String[] args) throws Exception {
+
+        robot = new Robot();
+        robot.setAutoWaitForIdle(true);
+        robot.setAutoDelay(200);
+        // Skipping NimbusLookAndFeel & GTKLookAndFeel as both are not supported for this feature - JDK-8075914
```
Comment (mrserb) [inline test/jdk/javax/swing/JSplitPane/4820080/JSplitPaneDragColorTest.java:71]: Please split the long lines using 80 chars per line.
Note that JDK-8075914 is closed so it is unclear why this is unsupported by Nimbus/GTK l&f.

Comment (manukumarvs) [reply test/jdk/javax/swing/JSplitPane/4820080/JSplitPaneDragColorTest.java:71]: I think the closed bug is JDK-8075608, but JDK-8075914 is still open. But anyway JDK-8075608 was closed as 'Won't Fix', so I think the problem remains.

Comment (manukumarvs) [reply test/jdk/javax/swing/JSplitPane/4820080/JSplitPaneDragColorTest.java]: Fixed the long lines issue.

**Review by mrserb — APPROVED** [review]
Looks fine, please confirm that mach5 is green before integration.

**Comment by manukumarvs** [comment]
> Looks fine, please confirm that mach5 is green before integration.

Confirmed with Mach5 execution in Linux,Mac and Windows machines(10 times run in each platform) and got all pass.

**Pull request closed without merging**